styles/odata.css

```diff
@@ -208,6 +208,17 @@ hr:first-of-type {
 }
 
 @media screen {
+  #darkmode {
+    position: fixed;
+    top: 0;
+    right: 0;
+  }
+
+  body.darkmode {
+    background-color: black;
+    color: white;
+  }
```

Comment:
The hardest part is still missing: What is a suitable style for dark mode? Should we ask the OASIS colleagues who provided us with the initial Markdown template?